packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -12,10 +12,11 @@ import '../../shared/screen.dart';
 import '../api/vs_code_api.dart';
 
 class DebugSessions extends StatelessWidget {
-  const DebugSessions(this.api, this.sessions, {super.key});
+  const DebugSessions(this.api, this.sessions, this.deviceMap, {super.key});
```

Comment:
nit: can we make these required and named so that constructing `DebugSessions` is more readable?

Reply:
Done!